lineman/app/modules/discussion_page/thread_card/thread_card.coffee

```diff
@@ -0,0 +1,40 @@
+angular.module('loomioApp').directive 'threadCard', ->
```

Comment:
I've gotten tired of the directive + controller separate files format we are using and feel like it would be nicer if we just had one coffee file per module, seeing as directives are pretty fixed length anyway.  This way we just have one coffee file per module. Whaddya think @gdpelican?